include/SFML/Window/Clipboard.hpp

```diff
@@ -43,27 +43,27 @@ class SFML_WINDOW_API Clipboard
 {
 public:
 
-	////////////////////////////////////////////////////////////
-	/// \brief Get the content of the clipboard as string data
-	///
-	/// This function returns the content of the clipboard
-	/// as a string.
-	///
-	/// \return Current content of the clipboard
-	///
-	////////////////////////////////////////////////////////////
-	static String getString();
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the content of the clipboard as string data
+    ///
+    /// This function returns the content of the clipboard
+    /// as a string.
+    ///
```

Comment:
At least on Windows there's some automatic conversion going on (and programs can intercept pasting to provide fitting data as well), but I'd at least add copying/pasting images, since it's one core part of SFML. Only problem i see is this adding a dependency back to sfml-graphics? Or should it just return the raw pixels in a vector?